py/tracks/latest.toml

```diff
@@ -13,6 +13,7 @@ fixtures = [
 queries = [
     "../specs/group_by.toml",
     "../specs/like.toml",
+    "../specs/regex.toml",
```

Comment:
if this is listed under queries it needs to have the same setup/teardown instructions as `../specs/group_by`.